core/src/main/java/cz/o2/proxima/repository/AttributeValueDescriptor.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2017-2021 O2 Czech Republic, a.s.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package cz.o2.proxima.repository;
+
+import cz.o2.proxima.scheme.SchemaDescriptors.ArrayTypeDescriptor;
+import cz.o2.proxima.scheme.SchemaDescriptors.PrimitiveTypeDescriptor;
+import cz.o2.proxima.scheme.SchemaDescriptors.SchemaTypeDescriptor;
+import cz.o2.proxima.scheme.SchemaDescriptors.StructureTypeDescriptor;
+import java.io.Serializable;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+/**
+ * Attribute value descriptor.
```

Comment:
Hm, this javadoc does not bring too much information. :-)
Can we either expand it somehow, or remove it?

Reply:
Hm, it's unclear to me what this class actually does. It only wraps `SchemaTypeDescriptor`, why we cannot simply use that?

Reply:
From my perspective `AttributeValueDescriptor` means more that just `scheme` that's why I create this class but I agree currently it is just a wrapper. Still not sure. Maybe just `getValueSchemeDescripor` on `AttributeDescriptor` should be enough. WDYT?

Reply:
Class completely removed.